public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -135,6 +138,18 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
     };
   }
 
+  /**
+   * Augment the results with links back to the azure console
+   */
+  query(request: DataQueryRequest<AzureMonitorQuery>): Observable<DataQueryResponse> {
+    return super.query(request).pipe(
+      map((res: DataQueryResponse) => {
+        // ???return from(this.processResponse(res)); // ???
```

Comment:
@torkelo -- thoughts on the best approach here?

Reply:
since processResponse also is async (does network calls), I think we need to make return an observable

Reply:
but pushed simplest fix for now, if you have a map that returns observable you need to use mergeMap